packages/atomic/stencil.config.ts

```diff
@@ -46,6 +46,7 @@ export const config: Config = {
       componentCorePackage: '@coveo/atomic',
       proxiesFile: '../atomic-react/src/components/stencil-generated/index.ts',
       includeDefineCustomElements: true,
+      excludeComponents: ['atomic-result-template', 'atomic-field-condition'],
```

Comment:
A bit unrelated, but maybe the atomic-external should be investigated and perhaps excluded too if it makes thing too complicated (the react context should remove the necessity for it)